randovania/games/dread/json_data/Artaria.txt

```diff
@@ -5197,22 +4841,24 @@ Extra - asset_id: collision_camera_081
               Walljump (Beginner) and Use Spin Boost
```

Comment:
This part can have walljump removed, so only simple ibj or use spin boost. and probably doesnt need the Before Rotatable. There is no danger if the rotatable has already been moved.